src/components/RatingAndReviews/index.jsx

```diff
@@ -1,19 +1,55 @@
-import React from 'react';
-import StarsRating from '../ReviewStars/StarsRating.jsx'; // eslint-disable-line
-import ProductStarRating from '../ReviewStars/ProductStarRating.jsx'; // eslint-disable-line
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import RatingSummary from './RatingSummary.jsx'; // eslint-disable-line
+import ReviewsList from './ReviewsList.jsx'; // eslint-disable-line
+import ReviewForm from './ReviewForm.jsx'; // eslint-disable-line
+import Modal from '../UI/Modal.jsx'; // eslint-disable-line
+
+const Wrapper = styled.div`
+  border: 1px solid;
+`;
+
+const SummaryAndListWrapper = styled.div`
+  display: flex;
+`;
+
+const BtnWrapper = styled.div``;
+
+const AddFormBtn = styled.button`
+  display: block;
+  margin: auto;
+`;
+
+function RatingAndReviews({ productId, productName = 'Anonymous' }) { // eslint-disable-line
+
+  const [showForm, setShowForm] = useState(false);
+  const openModal = () => {
+    setShowForm(true);
+  };
+  const closeModal = () => {
+    setShowForm(false);
+  };
+  const [showWidget, setShowWidget] = useState(false);
 
-function RatingAndReviews() {
   return (
     <>
-      <p>RatingAndReviews</p>
-      <StarsRating stars={3.7} />
-      <StarsRating stars={4.7} />
-      <StarsRating stars={1.7} />
-      <p>Review For ID 40344:</p>
-      <ProductStarRating productId={40344} />
-      <p>Review For ID 40347:</p>
-      <ProductStarRating productId={40347} />
-
+      <button type="button" onClick={() => { setShowWidget((cur) => !cur); }}>Toggle Lance&apos;s widget</button>
+      {showWidget && (
```

Comment:
I add a state to toggle the Whole Widget here. So you can close my widget while developing. Not seeing my ugly UI might be critical for a better Developer Experience.